SwiftWisdom/Core/StandardLibrary/Array/Sequence+Utilities.swift

```diff
@@ -13,7 +13,7 @@ extension Sequence {
 }
 
 extension Sequence {
-    public func ip_splitFilter(filter: @noescape (Iterator.Element) throws -> Bool) rethrows -> (passed: [Iterator.Element], failed: [Iterator.Element]) {
+    public func ip_splitWith(filter: (Iterator.Element) throws -> Bool) rethrows -> (passed: [Iterator.Element], failed: [Iterator.Element]) {
```

Comment:
get that 'with' in the parameter `ip_split(withFilter filter: (Iterator.Element) throws -> Bool)`